ddln_imaginator.proto

```diff
@@ -36,100 +55,119 @@ message Environment {
             STRATUS = 6;
         }
         optional Type type = 1;
-        optional float altitude_m = 2;
 
-        // density value: 0 - 1
+        // Cloud altitude in world coordinate frame [m]
+        optional float altitude = 2;
+
+        // density value, typical range 0..1 [-]
         optional float density = 3;
 
-        // directional x,y,z values to set the wind in the Engine.
+        // directional x, y, z values to set the wind in ENU coordinates system [m/s]
         optional Point3d velocity = 4;
     }
-    repeated Clouds clouds = 1;
+    repeated Clouds clouds = 2; // Up to 3 layers supported
```

Comment:
same: move to the line above

Reply:
nit: it is nice if all comments are full English sentences: starting with a capital letter, ending with a dot, contain a verb etc.